bitnami/cassandra/templates/headless-svc.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "cassandra.fullname" . }}-headless
+  labels:
```

Comment:
Be consistent with labels format. Use the same one you used on the secret definition.